addRemove.test.js

```diff
@@ -0,0 +1,29 @@
+import TodoList from './src/module/updateStatus.js';
```

Comment:
- [ ] Good job testing the add and delete files but you should also used mock for funtions that used localStorage, DOM or external APIs

Reply:
Hi @GzimAsani DOM and external API should be tested in the second part of the project. I have mentioned that in a comment.

Reply:
Hi, @Gegardus, sorroy for my mistake

Reply:
Hi @GzimAsani what we can do for now